### DISCUSSION TITLE
Extruder swap on 1st layer

### DISCUSSION THREAD

**Discussion by MakManIfest** [supermerill/SuperSlicer#3319] [Q&A]
Is there a way in SS to do a extruder swap on the first layer. I've tinkered and I can't see how to do it as SS considers the first layer as a support and enforces a single extruder to print it.

I only ask as I've seen stealthburner fronts (which is the 1st layer) with the perimeters a different colour. Perhaps this is done through post processing of the gcode?

**Comment by neophyl** [comment]
Might want to have a read of https://forum.prusa3d.com/forum/original-prusa-i3-mk3s-mk3-user-mods-octoprint-enclosures-nozzles/simple-way-to-print-in-real-multi-color-without-mmu-or-layer-height-based-changes/  (there are also several other threads on the prusa forum about printing different colours without different extuders or an mmu.

All the same techniques apply to SuSi.

Edit - Theres also https://forum.prusa3d.com/forum/prusaslicer/manual-multicolor/

**Reply by MakManIfest** [reply]
Thanks! I will have a read.
